Update CHANGELOG (#206)
chfast/ethash: CHANGELOG.md

# Changelog

## [0.8.0] — unreleased

> - Added: The new set of functions what verify Ethash hashes against difficulty
>   have been added to the library: `verify_against_difficulty()` and

Reviewer: ```suggestion
- Added: The new set of functions which verify Ethash hashes against difficulty
  has been added to the library: `verify_against_difficulty()` and
```
